documentation/docs/FAQ.mdx

```diff
@@ -46,58 +80,25 @@ OPAL solves the most pain for application-level authorization; where things move
 
    It may be possible to achieve a similar result with K8s Secrets, though there is no current way to do so with kube-mgmt. In addition this would tightly couple (potentially external) services into the k8s configuration.
 
-### Can I run the OPAL-server without `OPAL_BROADCAST_URI` ?
-
-<div
-  style={{
-    padding: "4px 8px",
-    backgroundColor: "#8573ff",
-    marginBottom: "10px",
-    width: "120px",
-    display: "flex",
-    alignItems: "center",
-    justifyContent: "center",
-    color: "white",
-    fontWeight: "bolder",
-    fontSize: "0.8rem",
-    borderRadius: "4px",
-  }}
->
-  Networking
-</div>
-
-#### Answer:
-
-Yes, you can potentially choose to run it as a single instance with a single worker in which case you don’t need the broadcaster channel. The broadcaster, AKA the backbone pub-sub, is used to sync between opal-server instances, one instance doesn't require syncing of course.
-This is mainly useful for light-workloads (Where a single worker is enough) and development environments (where you just don't want the hassle of setting up a Kafka / Redis / Postgres service)
+# Networking & Connectivity
 
-### Does OPAL provide health-check routes ?
-
-<div
-  style={{
-    padding: "4px 8px",
-    backgroundColor: "#8573ff",
-    marginBottom: "10px",
-    width: "340px",
-    display: "flex",
-    alignItems: "center",
-    justifyContent: "center",
-    color: "white",
-    fontWeight: "bolder",
-    fontSize: "0.8rem",
-    borderRadius: "4px",
-  }}
->
-  Devops, maintenance and observability
-</div>
-
-#### Answer:
+## Does OPAL handle reconnecting ?
 
-Yes both OPAL-server and OPAL-client serve a health check route at `"/healthcheck"`
+OPAL-clients constantly try to reconnect to their opal-servers, with an exponential back off.
+Once a client reconnects it will follow the data instructions in `OPAL_DATA_CONFIG_SOURCES` to again reach a baseline state for data.
+
+## How does OPAL handle data consistency / changes ?
+
+Short answer: with Speed.
+The pub/sub web socket is very lightweight and very fast - usually updates propagate within few milliseconds at worst.
+The updates themselves (by default) are also lightweight and contain only the event metadata (e.g. topic, data source), not the data itself.
+Hence OPAL-clients are aware of changes very quickly, and can react to the change (within their policy or healthchecks) even before they have the full data update.
+
+## Can I run the OPAL-server without `OPAL_BROADCAST_URI` ?
 
-### How does OPAL guarantee that the policy data is actually in sync with the actual application state. We get that OPAL server publishes updates to OPAL client but is there error correction in place.
+This is mainly useful for light-workloads (Where a single worker is enough) and development environments (where you just don't want the hassle of setting up a Kafka / Redis / Postgres service)
```

Comment:
```suggestion
Yes. This is mainly useful for light-workloads (Where a single worker is enough) and development environments (where you just don't want the hassle of setting up a Kafka / Redis / Postgres service).
Since server replication requires broadcasting, make sure that `UVICORN_NUM_WORKERS=1` and pod scaling is set to 1 (when running on k8s)
```